src/main/java/com/minecolonies/coremod/colony/requestsystem/requests/AbstractRequest.java

```diff
@@ -34,15 +35,15 @@
     @NotNull
     private final IToken<?>       token;
     @NotNull
-    private final R            requested;
+    private final R               requested;
     @NotNull
     private final List<IToken<?>> children;
     @NotNull
-    private final IRequester   requester;
+    private final IRequester      requester;
     @NotNull
     private RequestState state = RequestState.CREATED;
     @Nullable
-    private R      result;
+    private R         result;
     @Nullable
     private IToken<?> parent;
     @SuppressWarnings("squid:S1170")
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Suppressing the 'squid:S1170' warning is not allowed [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1309)